hazelcast/src/test/java/com/hazelcast/cluster/MulticastDeserializationTest.java

```diff
@@ -59,8 +63,17 @@
     // TTL = 0 : restricted to the same host, won't be output by any interface
     private static final int MULTICAST_TTL = 0;
 
-    @After
-    public void tearDown() {
+    @Rule
+    public OverridePropertyRule multicastGroupOverride = OverridePropertyRule
+            .clear(MulticastService.SYSTEM_PROPERTY_MULTICAST_GROUP);
+
+    @Before
+    public void before() {
+        cleanup();
```

Comment:
Looks like ClientDeserializationProtectionTest ran before this test and both tests are sharing a static instance(TestDeserialized) but ClientDeserializationProtectionTest does not set TestDeserialized.isDeserialized=false in an after method, so I guess it was already true when this test started.

I believe this line will fix the issue, we may also add TestDeserialized.isDeserialized=false to an after method of ClientDeserializationProtectionTest though.